### PR TITLE
Tk103 control output

### PR DESCRIPTION
**Pull request by parveenkumardeeva**
Tk103 Output Control command support added

### CODE REVIEW

File: test/org/traccar/protocol/Tk103ProtocolEncoderTest.java [hunk 1]
```diff
@@ -8,6 +8,21 @@
...
+    @Test
+    public void testEncodeOutputControl() {
+
+        Tk103ProtocolEncoder encoder = new Tk103ProtocolEncoder();
+
+        Command command = new Command();
+        command.setDeviceId(1);
+        command.setType(Command.TYPE_OUTPUT_CONTROL);
+        command.set(Command.KEY_INDEX, 0);
+        command.set(Command.KEY_DATA, "1");
```
Comment (tananaev) [inline test/org/traccar/protocol/Tk103ProtocolEncoderTest.java]: Is this the value we use for all other protocols?

Comment (parveenkumardeeva) [reply test/org/traccar/protocol/Tk103ProtocolEncoderTest.java]: It's referred from the CellocatorProtocolEncoderTest.
Is anything wrong here?

File: src/org/traccar/protocol/Tk103ProtocolEncoder.java [hunk 2]
```diff
@@ -97,6 +97,14 @@ protected Object encodeCommand(Command command) {
                     return formatCommand(command, "({%s}AV010)", Command.KEY_UNIQUE_ID);
                 case Command.TYPE_ENGINE_RESUME:
                     return formatCommand(command, "({%s}AV011)", Command.KEY_UNIQUE_ID);
+                case Command.TYPE_OUTPUT_CONTROL:
+                    if (command.getAttributes().containsKey(Command.KEY_DATA)) {
+                        if (command.getAttributes().get(Command.KEY_DATA).equals("1")) {
+                            return formatCommand(command, "({%s}AV001)", Command.KEY_UNIQUE_ID);
```
Comment (tananaev) [inline src/org/traccar/protocol/Tk103ProtocolEncoder.java]: I think it's better to just append `KEY_DATA` value to the command.

Comment (parveenkumardeeva) [reply src/org/traccar/protocol/Tk103ProtocolEncoder.java]: ok, Changing it.

**Comment by tananaev** [comment]
Merged, thanks.

**Pull request merged**